File: packages/cdktf-cli/templates/java/src/test/java/com/company/app/MainTest.java
import com.hashicorp.cdktf.Testing;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertTrue;

// The tests below are example tests, you can find more information at
// https://cdk.tf/testing
public class MainTest {

    @Test
    void myAppTest() {
        assertTrue(true);
    }
    //private final TerraformStack stack = new TerraformStack(Testing.app(), "stack");
    
    //private final MyApplicationsAbstraction appAbstraction = new MyApplicationsAbstraction(stack, "resource");
    //private final String synthesized = Testing.synth(stack);

    //@Test
    //void shouldContainContainer() {
    //    assertTrue(Testing.toHaveResource(synthesized, Container.TF_RESOURCE_TYPE) );
    //}

    //@Test
    //void shouldUseUbuntuImage() {
    //    assertTrue(Testing
    //            .toHaveResourceWithProperties(synthesized, Image.TF_RESOURCE_TYPE, new HashMap<String, Object>() {
    //                {
    //                    put("name", "ubuntu:latest");
    //                }
    //            }) );
    //}

    //@Test
    //void checkValidity() {
    //    assertTrue(Testing.toBeValidTerraform(Testing.fullSynth(stack)) );
    //}

}

File: packages/@cdktf/cli-core/templates/python-pip/main-test.py
import pytest
from cdktf import Testing, TerraformStack

# The tests below are example tests, you can find more information at
# https://cdk.tf/testing


class TestMain:

    stack = TerraformStack(Testing.app(), "stack")
    #app_abstraction = MyApplicationsAbstraction(stack, "app-abstraction")
    #synthesized = Testing.synth(stack)

    # def test_should_contain_container(self):
    #    assert Testing.to_have_resource(self.synthesized, Container.TF_RESOURCE_TYPE)

    # def test_should_use_an_ubuntu_image(self):
    #    assert Testing.to_have_resource_with_properties(self.synthesized, Image.TF_RESOURCE_TYPE, {
    #        "name": "ubuntu:latest",
    #    })

    # def test_check_validity(self):
    #    assert Testing.to_be_valid_terraform(Testing.full_synth(stack))
